migration/iD11/oracle/202304171928_IDEMPIERE-5567.sql

```diff
@@ -0,0 +1,10 @@
+-- IDEMPIERE-5567 Support of UUID on Search (FHCA-4195)
+SELECT register_migration_script('202304171928_IDEMPIERE-5567.sql') FROM dual;
+
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+ALTER TABLE t_selection MODIFY t_selection_uu DEFAULT ' ';
```

Comment:
hi Carlos, why is this prefer over making t_selection_uu nullable ?

Reply:
Hi @hengsin - because of the primary key of T_Selection table

Before it was AD_PInstance_ID + T_Selection_ID

To support this I needed to change it to AD_PInstance_ID + T_Selection_ID + T_Selection_UU

And T_Selection_UU being part of the primary key cannot be nullable.